nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/src/main/java/org/apache/nifi/processors/solr/SolrUtils.java

```diff
@@ -66,6 +67,15 @@
     public static final AllowableValue SOLR_TYPE_STANDARD = new AllowableValue(
             "Standard", "Standard", "A stand-alone Solr instance.");
 
+    public static final PropertyDescriptor RECORD_WRITER = new PropertyDescriptor

```

Comment:
I think this will work for now, but it would be great to have the Solr CRUD functions moved over to a controller service.

Reply:
This is a really nice idea. However, I would prefer to do this in a separate ticket.

Reply:
Ok. I might be able to help with some of that.